src/Microsoft.DotNet.Wpf/redist/VCRuntime/VCRuntime.vcxproj

```diff
@@ -33,8 +33,8 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <Import Project="$(WpfCppProps)" />
   <PropertyGroup>
-    <VCRedistCrtFolderName Condition="'$(Configuration)'=='Release'">Microsoft.VC142.CRT</VCRedistCrtFolderName>
-    <VCRedistCrtFolderName Condition="'$(Configuration)'=='Debug'">Microsoft.VC142.DebugCRT</VCRedistCrtFolderName>
+    <VCRedistCrtFolderName Condition="'$(Configuration)'=='Release'">Microsoft.VC143.CRT</VCRedistCrtFolderName>
```

Comment:
Is the fallback still necessary ?
https://github.com/dotnet/wpf/blob/af9ff74b557fb694f91944a609e71ed3e8b88568/src/Microsoft.DotNet.Wpf/redist/VCRuntime/VCRuntime.vcxproj#L38-L42

It seems weird to fallback to VC141 when VC143 does not exist.

Reply:
Not necessary @ThomasGoulet73 
Will be removed.

Reply:
@ThomasGoulet73 https://github.com/dotnet/wpf/pull/5810